suite/tests/win32/attachee.c

```diff
@@ -36,9 +36,7 @@
 int
 main(int argc, const char *argv[])
 {
-    for (int i = 0; i < 60; i++) {
-        Sleep(1000);
-    }
+    Sleep(3 * 60 * 1000);
```

Comment:
With it now working on infloop is the plan to delete this file?

Reply:
yes